validation/markets-validator.js

```diff
@@ -13,14 +13,17 @@
  */
 
 
+var assert = require('assert');
 var fs = require('fs');
 var path = require("path");
 var colors = require('colors');
 var opening_hours = require('opening_hours');
-var urlparse = require('url').parse;
 var http = require('http');
 var https = require('https');
+var pkg = require('../package.json');
 
+var REPO_URL = pkg.repository.url;
```

Comment:
```suggestion
// We're using the repository url as user-agent header in 
// url status validation (validateUrlStatus)
var REPO_URL = pkg.repository.url;
```

Reply:
:heavy_check_mark: Comment applied in 4268e1aaf72471053312dc523ac71dc645cf7908.